README.md

```diff
@@ -412,6 +421,75 @@ kubectl port-forward svc/marklogic 8000:8000
 
 This pod can now be accessed via http://localhost:8001.
 
+## HAPorxy Load Balancer
+
+HAProxy is provided as a load balancer that is configured to supported cookie-based session affinity and multi-statement transaction that is needed by some of the MarkLogic client application like MLCP. 
+
+### Enable HAProxy
+
+The HAProxy Load Balancer is disabled by default. To enable the HAProxy, provide the config the following in your values file for your Chart installation:
```

Comment:
should probably say "...provide the following config..."

Reply:
Fixed